packages/validate/src/lib/reducers/index.js

```diff
@@ -48,7 +48,7 @@ export default {
             state.groups[data.groupName]
                 ?  { validators: [...state.groups[data.groupName].validators, data.validator] }
                 : {
-                    fields: ([].slice.call(document.querySelectorAll('[data-val-'+GROUP_ATTRIBUTE+'="'+data.groupName+'"]'))) ? [].slice.call(document.querySelectorAll('[data-val-'+GROUP_ATTRIBUTE+'="'+data.groupName+'"]')) : [].slice.call(document.getElementsByName(data.groupName)),
+                    fields: document.querySelector('[data-val-'+GROUP_ATTRIBUTE+'="'+data.groupName+'"]') ? [].slice.call(document.querySelectorAll('[data-val-'+GROUP_ATTRIBUTE+'="'+data.groupName+'"]')) : [].slice.call(document.getElementsByName(data.groupName)),
```

Comment:
This was returning an empty array if no elements were found so it was always truthy. I didn't spot it at first but it caused a couple of tests to fail. It's fixed in this commit.